.github/workflows/rust.yml

```
@@ -70,3 +70,15 @@ jobs:
       - name: cargo quad-apk
         run: |
           cargo quad-apk build 
+
+  msrv:
+    name: MSRV Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)